specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5628 @@
+        "description": "Checks if the service has all the pre-requisites met to use a feature.",
+        "x-ms-examples": {
+          "services_getFeatureAvailibility": {
+            "$ref": "./examples/CheckFeatureAvailibility.json"
+          }
+        },
+        "operationId": "services_getFeatureAvailibility",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "featureName",
+            "in": "path",
+            "description": "The name of the feature.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/exporterrors/counts": {
+      "get": {
+        "tags": [ "Services" ],
+        "description": "Gets the count of latest AAD export errors.",
+        "x-ms-examples": {
+          "services_listExportErrors": {
+            "$ref": "./examples/ErrorCounts.json"
+          }
+        },
+        "operationId": "services_listExportErrors",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/ErrorCounts"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/exporterrors/listV2": {
+      "get": {
+        "tags": [ "Services" ],
+        "description": " Gets the categorized export errors.",
+        "x-ms-examples": {
+          "services_listExportErrorsV2": {
+            "$ref": "./examples/MergedExportErrors.json"
+          }
+        },
+        "operationId": "services_listExportErrorsV2",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "errorBucket",
+            "in": "query",
+            "description": "The error category to query for.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "nextLink",

```

Comment:
nextLink should not need to be exposed here.

Reply:
Yes you are right. I have removed it.